go.mod

```diff
@@ -1,15 +1,13 @@
-module github.com/zmap/zannotate
+module zannotate
```

Comment:
Was the `module` line changed intentionally? After this change I'm getting this error when trying to install zannotate.

```
# go install github.com/zmap/zannotate/cmd/zannotate@latest
go install github.com/zmap/zannotate/cmd/zannotate@latest: github.com/zmap/zannotate@none updating to
        github.com/zmap/zannotate@v0.0.0-20210922043249-9e938c460147: parsing go.mod:
        module declares its path as: zannotate
                but was required as: github.com/zmap/zannotate
```

I can successfully install the commit prior to this pull request.
```
# go install github.com/zmap/zannotate/cmd/zannotate@aae45de181c591172b9403d0d8d1984f9f71664e
```

I think the line should still be `module github.com/zmap/zannotate` unless I'm missing something or doing this wrong.

Reply:
I agree, this does seem like a regression. @DrewGregory would you be willing to submit a PR that reverts that one line?

Reply:
@ethack this should be resolved now that #29 is merged.

Reply:
That did it. Thank you!